frontend/src/components/adjudications/AdjudicationSection.jsx

```diff
@@ -65,14 +71,17 @@ export default function AdjudicationsSection(props) {
     history.push(`/events/${eventId}/adjudications/performance/${nextPerformance.id}`);
   };
 
+  var adjudicateNextButton = <Button variant="outlined" color="primary" onClick={goToNextPerformance}>Adjudicate Next</Button>;
+  if(isAdmin) {
```

Comment:
you can handle this with a ternary expression

`var adjudicateNext = isAdmin ? <></> : <Button variant ... > Adjudicate Next </Button>`